src/Components/Nav/Components/SignupModalEmail.js

```diff
@@ -1,145 +1,228 @@
-              <h3>생일</h3>
-              <p>
-                만 18세 이상의 성인만 회원으로 가입할 수 있습니다. 생일은 다른
-                에어비앤비 이용자에게 공개되지 않습니다.
-              </p>
-              <BirthSelect width="41.6%">
-                <option>월</option>
-                <option value="1">1월</option>
-                <option value="2">2월</option>
-              </BirthSelect>
-              <BirthSelect width="25%">
-                <option>일</option>
-                <option value="1">1</option>
-                <option value="2">2</option>
-              </BirthSelect>
-              <BirthSelect width="33.3%">
-                <option>년</option>
-                <option value="2020">2020</option>
-                <option value="2019">2019</option>
-              </BirthSelect>
-            </SignupInfoBirth>
+export default function SignupModalEmail({ handleSignupEmail }) {
+  const [email, setEmail] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [password, setPassword] = useState("");
+  const [successSignup, setSuccessSignup] = useState(false);
+  const [year, setYear] = useState("");
+  const [month, setMonth] = useState("");
+  const [day, setDay] = useState("");
+
+  const submitSignup = (e) => {
+    if (!email) return alert("이메일이 필요합니다.");
+    else if (!firstName) return alert("이름이 필요합니다.");
+    else if (!lastName) return alert("성을 입력하세요.");
+    else if (!password) return alert("비밀번호를 입력하세요.");
+
+    if (email && firstName && lastName && password) {
+      fetch(`${signInAPI}/user/signup`, {
+        method: "post",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          email: email,
+          first_name: firstName,
+          last_name: lastName,
+          password: password,
+          year: year,
+          month: month,
+          day: day,
+          platform: "airbnb",
+        }),
+      })
+        .then((res) => res.json())
+        .then((res) => {
+          if (res.message === "SUCCESS") {
+            alert("회원가입에 성공했습니다. 로그인 페이지로 넘어갑니다.");
+            setSuccessSignup(true);
+          } else {
+            alert("이메일 및 비밀번호를 확인해주세요");
+          }
+        });
+    }
+  };
+
+  return (
+    <ModalContainer>
+      {successSignup && <LoginModal />}
+      <ModalWrapper>
+        <ModalHeader>
+          <div onClick={() => handleSignupEmail()}>{NavSvg.modalClose}</div>
+          회원 가입
+          <div />
+        </ModalHeader>
+        <ModalContent>
+          <SocialSignup>
+            <ExtraAction noMargin>페이스북</ExtraAction>
+            또는
+            <ExtraAction noMargin>구글</ExtraAction>로 회원 가입하세요.
+          </SocialSignup>
+          <OrBorder>
+            <span>또는</span>
+          </OrBorder>
+          <InfoSection>
+            <div>
+              <InfoInput
+                onChange={(e) => {
+                  setEmail(e.target.value);
+                }}
```

Comment:
```suggestion
                onChange={(e) => setEmail(e.target.value)}
```
화살표 함수 축약형으로 쓰면 더 좋겠군요